dhis-2/dhis-services/dhis-service-expression-parser/src/main/resources/META-INF/dhis/beans.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-4.3.xsd">
+
+  <bean id="org.hisp.dhis.expressionparser.ExpressionParserService" class="org.hisp.dhis.expressionparser.DefaultExpressionParserService" />
+
+  <bean id="org.hisp.dhis.expressionparser.ExpressionItemsVisitor" class="org.hisp.dhis.expressionparser.ExpressionItemsVisitor" />
```

Comment:
Remove this mapping from beans.xml as this is never retrieved as a spring bean, rather instantiated in code.

Reply:
Done.